docs/scripts/make_model_zoo_docs.py

```diff
@@ -145,6 +163,17 @@
         prompt_field="frames.detections",  # can contain Detections or Keypoints
     )
 
+    session = fo.launch_app(dataset)
+{% elif 'med-sam' in name %}
```

Comment:
Suggestion: move this `elif` block above line 156 so that you can remove `and 'med-SAM' not in tags` from there?